packages/xod-tabtest/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.25.2",
   "scripts": {
     "refmt": "find src/ test/ -name '*.re*' -exec refmt --in-place {} +",
-    "build": "bsb -make-world",
+    "build:copy-catch2utils": "cpx \"../../cpplib/catch2utils/**\" \"./cpp/catch2utils\"",
```

Comment:
Seems like copying `XStringFormat.inl` should not be here, but rather be delivered in the same way as `catch2.hpp` does.

Such patterns are error-prone and lead to surprises when you see behavior changes only after full rebuilds.